packages/client/src/v2-events/features/events/Search/TabSearch.tsx

```diff
@@ -211,11 +211,31 @@ export function TabSearch({
     fieldValues
   })
 
-  const handleFieldChange = (fieldId: string, value: FieldValue) =>
+  const handleFieldChange = (fieldId: string, value: FieldValue) => {
+    const fieldTypesWithDefaults: FieldType[] = [FieldType.NAME]
```

Comment:
Is `fieldTypesWithDefaults` this benefiting us in some way?

Reply:
its a placeholder for any potential composite field component that requires default value to be populated with.

Reply:
It's not like if we just add the field type here, it would get automatically taken care of. Just below we need to do the manual check against the field type and handle the default value anyway. That's why it seems a bit redundant doesn't it?